docs/en/observability/synthetics-settings.asciidoc

```diff
@@ -0,0 +1,88 @@
+and send a notification using *connectors*.
+
+When you create a new synthetic monitor, a new default synthetics rule will be applied.
+To edit the default rule:
+
+. Click *Alerts and rules* in the top bar.
+. Click *Monitor status rule* to open a panel where you can edit the rule's configuration.
+
+To view all existing rules or create new rule:
+
+. Click *Alerts and rules* in the top bar.
+. Click *Manage rules*.
+
+Read more about creating new rules in <<create-alerts-rules>>.
+
+In the *Alerting* tab on the Synthetics Settings page, you can add and configure connectors.
+Read more about all available connectors in <<action-types-duration>>.
+
+[role="screenshot"]
+image::images/synthetics-settings-alerting.png[Alerting tab on the Synthetics Settings page in {kib}]
+
+[discrete]
+[[synthetics-settings-private-locations]]
+= Private locations
+
+Private locations allow you to run monitors from your own premises.
+In the *Private Locations* tab, you can add and manage private locations.
+
+After you <<synthetics-private-location-fleet-agent>> and <<synthetics-private-location-connect>>,
+this is where you will add the private location so you can specify it as the location for
+a monitor created using the Synthetics UI or projects.
+
+[role="screenshot"]
+image::images/synthetics-settings-private-locations.png[Private locations tab on the Synthetics Settings page in {kib}]
+
+[discrete]
+[[synthetics-settings-global-parameters]]
+= Global parameters
+
+Global parameters can be defined once and used across the configuration of browser and lightweight monitors.
+
+In the *Global parameters* tab, you can define variables and parameters.
+This is one of several methods you can use to define variables and parameters.
+To learn more about the other methods and which methods take precedence over others, see <<synthetics-params-secrets>>.
+
+[role="screenshot"]
+image::images/synthetics-settings-global-parameters.png[Global parameters tab on the Synthetics Settings page in {kib}]
+
+[discrete]
+[[synthetics-settings-data-retention]]
+= Data retention
+
+When you set up a synthetic monitor, data from the monitor is saved in {ref}/data-streams.html[Elasticsearch data streams],
+an append-only structure in Elasticsearch.
+You can customize how long synthetics data is stored by creating your own index lifecycle policy
+and attaching it to the relevant custom Component Template in Stack Management.
+
+In the *Data retention* tab, use the links to jump to the relevant policy for each data stream.
+Learn more about the data included in each data stream in <<synthetics-manage-retention>>.
+
+[role="screenshot"]
+image::images/synthetics-settings-data-retention.png[Data retention tab on the Synthetics Settings page in {kib}]
+
+[discrete]
+[[synthetics-settings-api-keys]]
+= Project API keys
+
+Project API keys are used to push monitors remotely from a CLI or CD pipeline.
```

Comment:
I'm not sure I like the repetition, but maybe it should be "...used to push _project_ monitors remotely..."?